agent/php_user_instrument.c

```diff
@@ -262,6 +244,12 @@ static nruserfn_t* nr_php_user_wraprec_create_named(const char* full_name,
   return wraprec;
 }
 
+static void wraprec_id_dtor(nruserfn_metadata *id) {
+  nr_free(id->filename.value);
```

Comment:
This seems to assume the `value` union member `cp` was set but there doesn't appear to be a way from the `zf_metadata_t` to actually verify if the data represented was allocated or not.  In this case you probably know the only way the `value` member is populated is using `strdup()` but maybe it would be good to have this info from the struct?

Reply:
You are correct - this code assumes that `cp` was set and at this point it should be set. However I will apply your suggestion to remove this assumption.

Reply:
Is ef8ebf666e0a17b49c8cde41492782371f5dce95 ok?